src/components/ScenarioParameters/UploadFileUtils.js

```diff
@@ -174,12 +175,46 @@ const resetUploadFile = (datasetId, file, setFile) => {
   }
 };
 
+function manageUploadFile (datasetId, file, fetchDataset, dataset, setDataset, setFile) {
```

Comment:
Could we rename this function into "updateDatasetState" ?
Also, I'm not sure this should be a part of the UploadFIleUtils, because its only goal is to update the state of the dataset in the ScenarioParameters component